docs/en/setup/README.md

```diff
@@ -1,11 +1,29 @@
 # Setup
 First and most important thing is, SkyWalking Satellite startup behaviours are driven by configs/satellite_config.yaml. Understanding the setting file will help you to read this document.
 
+## Requirements and default settings
+
+Requirement: **>= Go 1.14 are tested**. Other versions are not tested and may or may not work.
```

Comment:
Go binaries don't relies on Go version, this requirement is for development only I think.

Reply:
You're right, has been deleted.